### DISCUSSION TITLE
Status of checks in commit

### DISCUSSION THREAD

**Discussion by satishbkodali** [octokit/octokit.js#1782] [Q&A] [answered]
Is there any API that can tell me the status of checks corresponding to the commit in particular branch?

**Comment by gr2m** [comment, marked as answer]
I wrote a blog post about exactly that topic, it is not as straight forward as one would think:
https://dev.to/gr2m/github-api-how-to-retrieve-the-combined-pull-request-status-from-commit-statuses-check-runs-and-github-action-results-2cen

I think all your questions are answered in that blog post. Let me know if you have any further questions

**Reply by satishbkodali** [reply]
Thanks
